webapp/packages/core-blocks/src/CommonDialog/CommonDialog/CommonDialogHeader.tsx

```diff
@@ -46,9 +46,9 @@ export const CommonDialogHeader = observer<Props>(function CommonDialogHeader({
       <div className={s(computedStyles, { headerTitleContainer: true })}>
         <h3 className={s(computedStyles, { headerTitle: true })}>{translate(title)}</h3>
         {onReject && (
-          <div className={s(computedStyles, { reject: true })}>
-            <ActionIconButton aria-label={translate('ui_close')} name="cross" viewBox="0 0 16 16" onClick={onReject} />
-          </div>
+          <IconButton variant="secondary" size="small" aria-label="Close" onClick={onReject}>
+            <IconCross />
+          </IconButton>
```

Comment:
`IconButton` is a replacement for `ActionIconButton`, which is strange that (computedStyles, { reject: true })` is not passed to the `IconButton`. Especially because it has `flex-shrink: 0;`

Reply:
replacement is safe, styles are removed